x-pack/plugins/rollup/public/index_pattern_list/register.js

```diff
@@ -16,32 +16,28 @@ IndexPatternListRegistry.register(() => {
           key: 'rollup',
           name: 'Rollup',
         });
-
-        if(indexPattern.typeMeta && indexPattern.typeMeta.jobs) {
-          tags.push({
-            key: 'rollup_jobs',
-            name: `Rollup jobs: ${indexPattern.typeMeta.jobs.join(', ')}`,
-          });
-        }
       }
       return tags;
     }
 
     getFieldInfo = (indexPattern, field) => {
-      const jobs = indexPattern.typeMeta && indexPattern.typeMeta.jobs;
-      const capabilities = jobs && indexPattern.typeMeta.capabilities && indexPattern.typeMeta.capabilities[jobs[0]].fields[field];
+      const allAggs = indexPattern.typeMeta && indexPattern.typeMeta.aggs;
+      const fieldAggs = allAggs && Object.keys(allAggs).filter(agg => allAggs[agg][field]);
```

Comment:
Would `0` or an empty string be valid values for `allAggs[agg][field]` here? In which case we'd need to check against null/undefined instead of falsiness.

Reply:
nope, `allAggs[agg][field]` should be an array